modules/reference/pages/properties.adoc

```diff
@@ -0,0 +1,6361 @@
+= Redpanda Configuration Properties 
+:description: Redpanda configuration properties. 
+
+== Broker
+
+Broker configuration properties are applied individually to each broker in a cluster. 
+
+IMPORTANT: After you change a broker-level property setting, you must restart the broker for the change to take effect. 
+
+---
+
+=== admin
+
+Address and port of admin server
```

Comment:
```suggestion
Identifies the IP address and port of admin server.
```

Reply:
Global comment: @Deflaimun Let's be consistent with the use of upper/lowercase in the entries. I notice that requires restart has Yes/No, but visibility is user/tunable, etc. Nullable is Yes/No. So some inconsistency there.